pedidoRepository.js

```diff
@@ -1,10 +1,49 @@
 module.exports = function pedidoRepository(connection) {
   const get = async() => {
-    return await connection.query("SELECT TOP 50 número, data, hora FROM pedidos GROUP BY número,data, hora ORDER BY número DESC")
+    const query = `SELECT TOP 50 número, data, hora 
+                   FROM pedidos
+                   GROUP BY número, data, hora
+                   ORDER BY número DESC`
+
+    return await connection.query(query)
   }
 
   const getProdutos = async(numero) => {
-    return await connection.query(`SELECT número, produto, lançamento, cliente, vendedor, qte, valor, desconto, acrescimo, data, FormaPagto, hora, obs, cupom, DatadeEntrega, comissao, TipoPedido, StatusPedido FROM pedidos WHERE número = ${numero}`)
+    const query = `SELECT pe.número, pe.produto, pe.lançamento, pe.cliente,
+                          pe.vendedor, pe.qte, pe.valor, pe.desconto, pe.acrescimo, pe.data,
+                          pe.FormaPagto, pe.hora, pe.obs, pe.cupom, pe.DatadeEntrega, pe.comissao,
+                          pe.TipoPedido, pe.StatusPedido,
+                          pr.estoque_descrição
+                   FROM pedidos pe
+                   LEFT JOIN produtos pr ON pe.produto = pr.estoque_codigo
+                   WHERE número = ${numero}
+                   ORDER BY lançamento`
+
+    // first item is always the "pedido" summary, other items are the "produtos" from that "pedido"
+    const [pedido, ...produtos] = await connection.query(query)
```

Comment:
first item from array (lancamento = 0) is the pedidos summary, the others (..rest) are the produtos from the pedido